Logos/box.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="75" height="75" fill="#2a74ef"  xmlns:v="https://vecta.io/nano"><path fill-rule="evenodd" d="M28.731 37.078c1.846-3.479 5.512-5.848 9.724-5.848a11.01 11.01 0 0 1 11.007 11.006 11.01 11.01 0 0 1-11.007 11.01c-4.212 0-7.878-2.372-9.724-5.847-1.847 3.475-5.51 5.847-9.725 5.847-6.02 0-10.907-4.83-11.004-10.826H8V24.158a2.2 2.2 0 0 1 4.399 0v9.274a10.98 10.98 0 0 1 6.607-2.201c4.215 0 7.879 2.369 9.725 5.848zm-16.332 5.159c0 3.647 2.96 6.601 6.607 6.601a6.6 6.6 0 1 0 0-13.202c-3.647 0-6.607 2.954-6.607 6.6zm19.45-.002c0 3.648 2.958 6.603 6.606 6.603s6.602-2.955 6.602-6.603-2.956-6.599-6.602-6.599-6.606 2.954-6.606 6.599z"/><path d="M60.558 42.221l5.983 7.339c.759.973.541 2.323-.505 3.046s-2.515.539-3.326-.404l-5.147-6.308-5.149 6.308c-.802.942-2.28 1.127-3.323.404s-1.262-2.073-.501-3.046h-.002l5.98-7.339-5.98-7.353h.002c-.761-.97-.543-2.324.501-3.044 1.044-.726 2.522-.54 3.323.401v-.001l5.149 6.316 5.154-6.315c.81-.941 2.279-1.127 3.325-.401 1.046.719 1.263 2.074.505 3.044l-5.99 7.353z"/></svg>
```

Comment:
Image is not render properly in [svgLint](https://aka.ms/unicorns/svglint). X of a box getting truncated.

Please find the screenshot below:
![image](https://user-images.githubusercontent.com/73637076/108725228-2abe2200-754c-11eb-86a8-6f9551c3628c.png)


Thanks,
Aditi Maheshwari